src/coreclr/debug/daccess/task.cpp

```diff
@@ -5225,7 +5225,7 @@ EnumMethodInstances::Next(ClrDataAccess* dac,
         }
     }
 
-    if (!m_methodIter.Current()->HasNativeCodeReJITAware())
+    if (!(g_pEEInterface->GetFunctionAddress(m_methodIter.Current()) != NULL))
```

Comment:
Right now this is testing if the method has a default or active version with code. I suspect the best behavior is to test if there is any NativeCodeVersion that contains code, regardless whether it is default or active.

Reply:
Iterating through the DJIs and finding the first one that is not null? Is that what you mean?